### DISCUSSION TITLE
Hackaton :))))

### DISCUSSION THREAD

**Discussion by GIORGIOGAL** [Launch-X-Latam/MisionFrontEnd#538] [Team Posts]
Hola, buen día. Estoy buscando equipo para el hackaton, tengo experiencia como desarrollador de software y también en frontend. Además sé ocupar varios programas de diseño de Adobe :)))

**Comment by GIORGIOGAL** [comment]
mi correo : lx15999@innovaccion.mx

**Comment by snowingt** [comment]
Hola escribeme 
snowingtorres@gmail.com

**Comment by mafer23** [comment]
https://discord.gg/QZa2UAHu aqui se estan creando equipos con canal privado
, si gustan

**Comment by AbnerHub** [comment]
Hola, igual estoy en busca este es mi corre, espero puedas agregarme lx02522@innovaccion.mx

**Comment by rpajares** [comment]
Pues si tienes experiencia, este curso es para principiantes, deja que los
que estan aprendiendo participen sin ayuda de alguien que no necesita.

Le mer. 9 nov. 2022, 19 h 39, GIORGIOGAL ***@***.***> a
écrit :

**Comment by Tintto** [comment]
Hola ya conseguiste grupo? nosotros si valoramos tu experiencia !!

**Comment by Tintto** [comment]
si quieres escríbeme a lx11061@innovaccion.mx

**Comment by ErikaBarrera01** [comment]
hola!
 ya estan completos?

**Comment by GustavoLoreto** [comment]
Hola estoy disponible para equipo, tengo un poco de experiencia con front y back este es mi correo Gracias!
 lx15700@innovaccion.mx

**Comment by AbnerHub** [comment]
Hola igual estoy buscando equipo lx02522@innovaccion.mx

**Comment by IsraelVera** [comment]
Yo igual busco equipo ***@***.***

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: GIORGIOGAL ***@***.***>
Sent: Wednesday, November 9, 2022 6:40:07 PM
Cc: Subscribed ***@***.***>
Subject: [Launch-X-Latam/mision-backend] Hackaton :)))) (#72)


Hola, buen día. Estoy buscando equipo para el hackaton, tengo experiencia como desarrollador de software y también en frontend. Además sé ocupar varios programas de diseño de Adobe :)))

**Comment by Tfrnk** [comment]
Buenas tardes con todos alguien sabe donde puedo ver las transmisiones de
las clases ya que desde hace una semana ya no me sale
